src/flamenco/runtime/program/fd_vote_program.c

```diff
@@ -1303,42 +1359,53 @@ process_new_vote_state( fd_vote_state_t *           vote_state,
   }
 
   // Comment:
-  // https://github.com/firedancer-io/solana/blob/v1.17.5/programs/vote/src/vote_state/mod.rs#L703-L709
+  // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L736
   if( FD_LIKELY( timely_vote_credits ) ) {
+    // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L737
     for( deq_fd_landed_vote_t_iter_t iter = deq_fd_landed_vote_t_iter_init( new_state );
          !deq_fd_landed_vote_t_iter_done( new_state, iter );
          iter = deq_fd_landed_vote_t_iter_next( new_state, iter ) ) {
       fd_landed_vote_t * new_vote = deq_fd_landed_vote_t_iter_ele( new_state, iter );
+      // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L738
       if( FD_UNLIKELY( new_vote->latency == 0 ) ) {
         // this is unlikely because as validators upgrade, it should converge to the new vote state
+        // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L739
         new_vote->latency = compute_vote_latency( new_vote->lockout.slot, current_slot );
       }
     }
   }
 
   // doesn't matter what the value of slot if `is_some = 0` i.e. `Option::None`
+  // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L744
   int both_none = !vote_state->has_root_slot && !has_new_root;
   if( ( !both_none && ( vote_state->has_root_slot != has_new_root ||
                         vote_state->root_slot != new_root ) ) ) {
     increment_credits( vote_state, epoch, earned_credits );
   }
+  // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L750
   if( FD_LIKELY( timestamp != NULL ) ) {
     /* new_state asserted nonempty at function beginning */
     ulong last_slot = deq_fd_landed_vote_t_peek_tail( new_state )->lockout.slot;
     rc              = process_timestamp( vote_state, last_slot, *timestamp, ctx );
     if( FD_UNLIKELY( rc ) ) { return rc; }
     vote_state->last_timestamp.timestamp = *timestamp;
   }
+
+  // https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L754
   vote_state->has_root_slot = (uchar)has_new_root;
   vote_state->root_slot     = new_root;
-  // TODO can prob just fd_memcpy
+
+#if 0
+  // WTF
```

Comment:
I cannot find this code anywhere... went all the way back to 1.14.x ... why was this here?!

Reply:
https://github.com/anza-xyz/agave/blob/v2.0.0/programs/vote/src/vote_state/mod.rs#L755